presto-main/src/main/java/com/facebook/presto/execution/PageBufferInfo.java

```diff
@@ -26,6 +26,7 @@
     private final long bufferedPages;
     private final long queuedPages;
     private final long bufferedBytes;
+    private final long rowsAdded;
```

Comment:
The commit message doesn't seem to match what this is tracking (i.e., rows added to the buffer vs rows sent to the client)